### DISCUSSION TITLE
Why isn't MySQL 8 supported?

### DISCUSSION THREAD

**Discussion by MarkJacksonRG** [fluentmigrator/fluentmigrator#1669] [Q&A] [answered]
Hi, 
I want to use some simple migrations with MySql 8. I see version 8 is not supported according to the [list of supported databases](https://fluentmigrator.github.io/) on the documentation site. The list only mentions version 4 and 5.

I'm using Fluent Migrations to setup some simple databases in MySql and other database engines. Can I ask:

 - Why isn't MySql 8 supported? 
 - What issues are there with MySql 8? 

Thanks in advance!
Mark

**Comment by jzabroski** [comment, marked as answer]
You can always use the lowest version we support and it should automatically work unless there are breaking changes.

We don't have an explicit maintainer for each of the supported databases, so we only make changes when people encounter problems due to breaking changes. The main place this has seen effects is with recent changes to Sqlite and Postgres.
